CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 2.2.3
 
+### Enhancements
+
+* Exceptions thrown in error handler are not swallowed (#3559).
```

Comment:
This is a sync change, so should be in the sync section of the changelog. Perhaps "Exceptions thrown in the `SyncSession.ErrorHandler` are no longer swallowed by JNI" Would be more clear?